atlasdb-commons/src/main/java/com/palantir/common/collect/IteratorUtils.java

```diff
@@ -69,12 +69,7 @@
      * </code>
      */
     public static <T> Iterable<T> forEach(final Iterator<T> it) {
```

Comment:
Hmm, ok. If you're happy to replace `IteratorUtils.forEach(x)` with `() -> x` in $internalCodebase, then sure.